src/app/unlocked-stage/unlocked-stage.component.html

```diff
@@ -0,0 +1,59 @@
+<div id="container" [hidden]="gameOver || success">
+    <div id="scene">
+        <div id="human-health-text">{{language?.legend.human}}</div>
+        <div id="human-health">
+            <div id="human-health-bar"></div>
+        </div>
+        <div id="mosquito-health-text">{{language?.legend.mosquito}}</div>
+        <div id="mosquito-health">
+            <div id="mosquito-health-bar"></div>
+        </div>    
+        <img id="human" src="../assets/img/unlocked-stage/human.png"/>
+        <img id="moz" name="animate" src="../assets/img/unlocked-stage/mosquito.gif"/>  
+        <img id="spray" src="../assets/img/unlocked-stage/spray-cursor.gif"/>
+    </div>
+    <div id="inventory">
+        <div id="inventory-heading">{{language?.legend.inventory}}</div>
+        <img id="spray-item" src="../assets/img/unlocked-stage/spray-item.gif"/>
+    </div>
+</div>
+<div class="row col-lg-6 col-lg-offset-3 text-center" id="game-end-status" [hidden]="!gameOver">
+    <div class="swal2-icon swal2-error swal2-animate-error-icon">
+        <span class="swal2-x-mark swal2-animate-x-mark">
+            <span class="swal2-x-mark-line-left"></span>
+            <span class="swal2-x-mark-line-right"></span>
+        </span>
+    </div>
+    <div>
+        <h2 class="swal2-title text-center">{{language?.alerts.loss.title}}</h2>
```

Comment:
No.
The text is not centered if .text-center is removed.
Its a bootstrap class actually.